recordablesurfaceview/src/main/java/com/uncorkedstudios/android/view/recordablesurfaceview/RecordableSurfaceView.java

```diff
@@ -251,13 +253,58 @@ public void initRecorder(File saveToFile, int displayWidth, int displayHeight,
         mediaRecorder.setAudioSamplingRate(44100);
         mediaRecorder.setAudioEncodingBitRate(96000);
 
+        mediaRecorder.setVideoEncoder(MediaRecorder.VideoEncoder.DEFAULT);
+
+        mediaRecorder.setVideoEncodingBitRate(12000000);
+        mediaRecorder.setVideoFrameRate(30);
+        mediaRecorder.setVideoSize(displayWidth, displayHeight);
+
+        mediaRecorder.setOutputFile(saveToFile.getPath());
+        mediaRecorder.prepare();
+
+        mMediaRecorder = mediaRecorder;
+    }
+
+    /**
+     * Iitializes the {@link MediaRecorder} ad relies on its lifecycle and requirements.
+     *
+     * @param saveToFile    the File object to record into. Assumes the calling program has
+     *                      permission to write to this file
+     * @param displayWidth  the Width of the display
+     * @param displayHeight the Height of the display
+     * @param orientationHint the orientation to record the video (0, 90, 180, or 270)
+     * @param errorListener optional {@link MediaRecorder.OnErrorListener} for recording state callbacks
+     * @param infoListener  optional {@link MediaRecorder.OnInfoListener} for info callbacks
+     * @see MediaRecorder
+     */
+    @SuppressWarnings({"all"})
+    public void initRecorder(File saveToFile, int displayWidth, int displayHeight,
```

Comment:
I had originally called through to this method from the other initRecorder with a value of 0 for orientationHint. However, I was concerned that this would not behave correctly on all devices. Perhaps a builder approach makes sense?

Reply:
I think I was worried about nothing.